src/block-fly/display/messageDisplay.ts

```diff
@@ -0,0 +1,14 @@
+import "material-design-lite";
+
+export function showMessage(message: string): void {
+  const messageContainer = document.getElementById("message") as any;
+
+  componentHandler.upgradeElement(messageContainer);
+
+  const data = {
+      message: message,
+      timeout: 2000
```

Comment:
2 seconds doesn't seem like enough for errors. Maybe add optional timeout parameter and put it longer for errors?